framework/src/Volo.Abp.Auditing/Volo/Abp/Auditing/JsonNetAuditSerializer.cs

```diff
@@ -18,7 +18,7 @@ public string Serialize(object obj)
         {
             var options = new JsonSerializerSettings
             {
-                ContractResolver = new AuditingContractResolver(Options.IgnoredTypes)
+                ContractResolver = AuditingContractResolver.GetSharedAuditingContractResolver(Options.IgnoredTypes)
```

Comment:
Actually, Options.IgnoredTypes are not so dynamic. You can lazy-create AuditingContractResolver and reuse it without caring Options.IgnoredTypes changes later.

Reply:
The `AuditingContractResolver `is preferably static.
Although Lazy is reused to a certain extent, But still not as good as the global static `AuditingContractResolver`.

Reply:
You can completely remove AuditingContractResolver and move this code to JsonNetAuditSerializer.cs. What do you think?

Reply:
Already moved.